pkg/security/security_profile/profile/profile.go

```diff
@@ -80,16 +83,19 @@ func NewSecurityProfile(selector cgroupModel.WorkloadSelector, anomalyDetectionE
 		selector:               selector,
 		anomalyDetectionEvents: anomalyDetectionEvents,
 		eventTypeState:         make(map[model.EventType]*EventTypeState),
+		anomaliesCpt:           atomic.NewUint64(0),
```

Comment:
Can we spell out cpt cuz I don't know what you mean here.

Reply:
Yep, I'll change that to `anomaliesCounter` (yep.. "cpt" was a french abbreviation, my bad!)